authorization-service/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "authorization-service",
+  "version": "1.0.0",
+  "description": "Serverless aws-nodejs-typescript template",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "deploy": "yarn sls deploy",
+    "migrate": "ts-node node_modules/.bin/node-pg-migrate -j ts",
+    "migrate:up": "yarn migrate up",
+    "migrate:down": "yarn migrate down"
+  },
+  "engines": {
+    "node": ">=14.15.0"
+  },
+  "dependencies": {
+    "@babel/preset-typescript": "^7.14.5",
+    "@middy/core": "^1.5.2",
+    "@middy/http-cors": "^2.4.1",
+    "@middy/http-json-body-parser": "^1.5.2",
+    "@types/jest": "^26.0.23",
```

Comment:
Looks like we can move some `dependencies` to the `devDependencies` instead. And it seems there are some redundant dependencies such as `pg`

Reply:
Yep, agree. I tried to handle these dependencies, to remove modules not related to shared module, but got webpack errors and give up without resolving :P